### DISCUSSION TITLE
Welcome to Bitcoin Pulse Discussions!

### DISCUSSION THREAD

**Discussion by ZiyadBouazara** [ZiyadBouazara/bitcoin-pulse#7] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by dirnalafeinte** [comment]
I’m excited to be part of the Bitcoin Pulse community! I’ve recently come across this project, and I’m really interested in how it tracks and monitors Bitcoin activity.

**Comment by samiusss** [comment]
Wow such an interesting project, I'm delighted to contribute to this work 🚀🚀
